### ISSUE TITLE
ag-grid and radio button conflict

### ISSUE THREAD

**Issue by jomosi** [elimintz/justpy#129]
radio button with ag-grid dosen't seem to work ....
could you please check out following sample codes?

--------------------------------------------
import justpy as jp
import pandas as pd

def radio_test():
    wp = jp.WebPage()

    df = pd.DataFrame([[1,2], [3,4]], columns=['a', 'b'])
    g = df.jp.ag_grid(a=wp)

    genders = ['male', 'female', 'other']
    ages = [(0, 30), (31, 60), (61, 100)]

    outer_div = jp.Div(classes='border m-2 p-2 w-64', a=wp)

    jp.P(a=outer_div, text='Please select your gender:')
    for gender in genders:
        label = jp.Label(classes='inline-block mb-1 p-1', a=outer_div)
        radio_btn = jp.Input(type='radio', name='gender', value=gender, a=label)
        jp.Span(classes='ml-1', a=label, text=gender.capitalize())

    jp.Div(a=outer_div, classes='m-2')  # Add spacing and line break
    jp.P(a=outer_div, text='Please select your age:')
    for age in ages:
        label = jp.Label(classes='inline-block mb-1 p-1', a=outer_div)
        radio_btn = jp.Input(type='radio', name='age', value=age[0], a=label)
        jp.Span(classes='ml-1', a=label, text=f'{age[0]} - {age[1]}')
        jp.Br(a=outer_div)

    return wp

jp.justpy(radio_test)

**Comment by elimintz** [comment]
Thank you for flagging this bug! It occurs with charts also, not just grids. The handling of radio buttons is thrown off when one of the elements on the page does not have a `components` attribute. 

I fixed it for the next version but with the current versions there is a simple workaround. Just add the line:
`g.components = []`
after the grid is defined.

This is the revised example with the line added:
```python
import justpy as jp
import pandas as pd

def radio_test():
    wp = jp.WebPage()

    df = pd.DataFrame([[1,2], [3,4]], columns=['a', 'b'])
    g = df.jp.ag_grid(a=wp)

    # Add this line
    g.components = []

    genders = ['male', 'female', 'other']
    ages = [(0, 30), (31, 60), (61, 100)]

    outer_div = jp.Div(classes='border m-2 p-2 w-64', a=wp)

    jp.P(a=outer_div, text='Please select your gender:')
    for gender in genders:
        label = jp.Label(classes='inline-block mb-1 p-1', a=outer_div)
        radio_btn = jp.Input(type='radio', name='gender', value=gender, a=label)
        jp.Span(classes='ml-1', a=label, text=gender.capitalize())

    jp.Div(a=outer_div, classes='m-2')  # Add spacing and line break
    jp.P(a=outer_div, text='Please select your age:')
    for age in ages:
        label = jp.Label(classes='inline-block mb-1 p-1', a=outer_div)
        radio_btn = jp.Input(type='radio', name='age', value=age[0], a=label)
        jp.Span(classes='ml-1', a=label, text=f'{age[0]} - {age[1]}')
        jp.Br(a=outer_div)

    return wp

jp.justpy(radio_test)
```

Let me know if you still encounter a problem.

**Comment by jomosi** [comment]
A workaround you've given is working well, Thanks!
And I am looking forward to new version of JUSTPY. :-)

**Comment by elimintz** [comment]
The bug was fixed in version 0.1.4
